src/services/shims.ts

```diff
@@ -72,18 +72,28 @@ namespace ts {
         directoryExists(directoryName: string): boolean;
     }
 
-    /** Public interface of the the of a config service shim instance.*/
-    export interface CoreServicesShimHost extends Logger, ModuleResolutionHost {
+    /** Public interface of the core-services host instance used in managed side */
+    export interface CoreServicesShimHost extends Logger {

```

Comment:
most of these members were inherited from the `ModuleResolutionHost`. Is it really necessary to duplicate them?